declarations/Amazon.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Amazon",
+  "documents": {
+    "Developer Terms": {
+      "fetch": "https://developer.amazon.com/support/legal/alexa-hipaa-baa",
+      "select": [
+        ".online-agreement-system-document > div > div"
+      ],
+      "remove": [
+        "em"
```

Comment:
This is too broad a selector for removing the last “[Remainder of Page Intentionally Left Blank]” (which, in my view, could even be tracked without it being a problem if finding a reliable selector is too hard).

Reply:
Agreed, I removed